source/backend/dal/Repositories/ProjectRepository.cs

```diff
@@ -74,28 +74,35 @@ public Task<Paged<PimsProject>> GetPageAsync(ProjectFilter filter)
         /// </summary>
         /// <param name="projectId"></param>
         /// <returns></returns>
-        public async Task<PimsProject> Get(long projectId)
+
+        public PimsProject Get(long id)
         {
             User.ThrowIfNotAuthorized(Permissions.ProjectView);
 
-            return await Context.PimsProjects
+            return Context.PimsProjects
                     .AsNoTracking()
+                    .Include(x => x.PimsProducts)
                     .Include(x => x.ProjectStatusTypeCodeNavigation)
                     .Include(x => x.RegionCodeNavigation)
-                    .Where(x => x.Id == projectId)
-                    .FirstOrDefaultAsync();
+                    .Where(x => x.Id == id)
+                    .FirstOrDefault();
```

Comment:
same comment here, I think you removed the null checking on the controller but I worry that the mapper doesn't handle nulls.

Reply:
no problem if null